src/components/NumberInput/NumberInput.js

```diff
@@ -53,7 +53,7 @@ export default class NumberInput extends Component {
         {
           value: evt.target.value,
         },
-        evt => {
+        () => {
```

Comment:
having `evt` here was overriding the `evt` passed to the function. The callback to setState doesn't pass anything so evt became undefined